amundsen_application/static/js/components/common/SearchBar/index.tsx

```diff
@@ -108,13 +138,32 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     return true;
   };
 
+  onSelectInlineResult = (resourceType: ResourceType, updateUrl: boolean = false) : void => {
+    this.hideTypeAhead();
+    this.props.onSelectInlineResult(resourceType, this.state.searchTerm, updateUrl);
+  }
+
+  shouldShowTypeAhead = (searchTerm: string) : boolean => {
+    return searchTerm.length > 0;
+  }
+
+  updateTypeAhead = (event: Event): void => {
+    /* This logic will hide/show the inline results component when the user clicks
+      outside/inside of the search bar */
+    if (this.refToSelf.current.contains(event.target as Node)) {
```

Comment:
I'm actually not sure. I think it could technically be null at specific points in the lifecycle either before the component is render or while it's re-rendering. So I think the case is that `refToSelf.current` *can* be null, but it would not be possible for it to be null in the specific context in which this method is used. That being said, I will add a check here anyway to be sure the app won't crash.